### PR TITLE
fix: fixing text overlap with minimum width of selects (M2-7753)

### PR DESCRIPTION
**Pull request by ramirlm**
### 📝 Description

🔗 [Jira Ticket M2-7753](https://mindlogger.atlassian.net/browse/M2-7753)

Changes include:

- Adds a `minWidth` to prevent overlap

### 📸 Screenshots

| Before (Optional)                      | After                                 |
| -------------------------------------- | ------------------------------------- |
CleanShot 2024-10-29 at 19 49 19@2x  | Code 2024-10-29 19 46 28  |

### 🪤 Peer Testing

𝐒𝐭𝐞𝐩𝐬 𝐭𝐨 𝐫𝐞𝐩𝐫𝐨𝐝𝐮𝐜𝐞:
Start creating an applet
Create any activity with all item types
Go to the Item flow tab
Select Time Range Item in a rule row in Conditional 1
Select start and end time
Pay attention to the screen

𝐀𝐜𝐭𝐮𝐚𝐥 𝐫𝐞𝐬𝐮𝐥𝐭: Text overlay at the rule row after Condition adding
𝐄𝐱𝐩𝐞𝐜𝐭𝐞𝐝 𝐫𝐞𝐬𝐮𝐥𝐭: No text overlay at the rule row after Condition adding
𝐀𝐝𝐝𝐢𝐭𝐢𝐨𝐧𝐚𝐥 𝐢𝐧𝐟𝐨𝐫𝐦𝐚𝐭𝐢𝐨𝐧: 
Relevant for empty condition or any selected item

### CODE REVIEW

**Review by LashaunnaS — COMMENTED** [review]
This looks so much better off the hop 🙌🏾 Do we have a specific min-width we're targeting on this project, it looks like we're getting a bit of overflow on small - mid size screens? 

Screenshot 2024-10-30 at 15 55 06

**Comment by ramirlm** [comment]
> This looks so much better off the hop 🙌🏾 Do we have a specific min-width we're targeting on this project, it looks like we're getting a bit of overflow on small - mid size screens?
> 
> Screenshot 2024-10-30 at 15 55 06

Thank you for your comment @LashaunnaS ! Maybe @hotavocado can help on that one?

**Comment by ramirlm** [comment]
@ChaconC - re-added 
https://github.com/ChildMindInstitute/mindlogger-admin/tree/feature/M2-7638-items-flow--summary-row-show-all

File: src/modules/Builder/components/ItemFlowSelectController/ItemFlowSelectController.styles.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export const StyledSelectController = styled(SelectController)({
 
   '&& .MuiSelect-select': {
     padding: theme.spacing(0.65, 1.2),
+    minWidth: '14rem',
```
Comment (ChaconC) [inline src/modules/Builder/components/ItemFlowSelectController/ItemFlowSelectController.styles.ts]: Now that you added `width: 100%;` to `StyledTextField` in `SelectController` I believe this is no longer necessary and will address @LashaunnaS' comment about smaller screens

| 14rem | 100% |
|-|-|
|CleanShot 2024-10-30 at 16 07 45|CleanShot 2024-10-30 at 16 07 32|

Comment (andrevitalb) [reply src/modules/Builder/components/ItemFlowSelectController/ItemFlowSelectController.styles.ts]: Nice spotting

Comment (ramirlm) [reply src/modules/Builder/components/ItemFlowSelectController/ItemFlowSelectController.styles.ts]: Indeed! Removed

**Comment by ramirlm** [comment]
> This looks so much better off the hop 🙌🏾 Do we have a specific min-width we're targeting on this project, it looks like we're getting a bit of overflow on small - mid size screens?
> 
> Screenshot 2024-10-30 at 15 55 06

Regarding that change, If you check the top nav menu it also hides a bit in smaller resolutions @LashaunnaS – so maybe we can create a future task to make the whole application more responsive for smaller resolutions if necessary :)

**Review by ChaconC — APPROVED** [review]

**Pull request merged**